From 4dd7a346193d13547b0eba0e93caeeddc5d4c7cb Mon Sep 17 00:00:00 2001
From: Adrian DC <radian.dc@gmail.com>
Date: Sun, 17 Jun 2018 12:38:23 +0200
Subject: [PATCH 26/28] SystemUI: Resolve status bar battery percentage tints

 * Use the tint color matching the status bar texts

 * Results: Before and after: https://ibb.co/gvGm7y

 * Test: Bright wallpaper with status bar contrasts:
    https://ibb.co/dxupSy

Change-Id: Idff1060af9e0a0fc5eab3ed498b10266e3124950
---
 .../SystemUI/src/com/android/systemui/BatteryMeterView.java     | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/packages/SystemUI/src/com/android/systemui/BatteryMeterView.java b/packages/SystemUI/src/com/android/systemui/BatteryMeterView.java
index 2b31967cea4..1327f4615eb 100644
--- a/packages/SystemUI/src/com/android/systemui/BatteryMeterView.java
+++ b/packages/SystemUI/src/com/android/systemui/BatteryMeterView.java
@@ -267,7 +267,7 @@ public class BatteryMeterView extends LinearLayout implements
         int background = getColorForDarkIntensity(intensity, mLightModeBackgroundColor,
                 mDarkModeBackgroundColor);
         mDrawable.setColors(foreground, background);
-        setTextColor(foreground);
+        setTextColor(tint);
     }
 
     public void setTextColor(int color) {
-- 
2.17.1

